setup.py

```diff
@@ -38,7 +38,7 @@
         'appdirs',
         'BlazeUtils',
         'blinker',
-        'Click>=3.0',
+        'Click<5.0',
```

Comment:
You may want "Click<=4.999"

otherwise it's possible to get RC / beta / etc. releases which are technically < 5.0.  Although, now that I think about it, the newer releases of pip exclude development releases by default, so it may not be a problem anymore.  Please check into this if you aren't sure one way or the other.